shared/src/simulation/events/vehicles-sent.ts

```diff
@@ -13,11 +14,24 @@ export class VehiclesSentEvent implements SimulationEvent {
     @ValidateNested()
     readonly vehiclesSent: VehicleResource;
 
+    @IsUUID(4, uuidValidationOptions)
+    readonly transferPointDestinationId: UUID;
```

Comment:
I think `destinationTransferPointId` would make more sense since the variable contains the ID of the transfer point that is the destination and not the ID of the destination of the transfer point (a transfer point does not move and therefore has no destination...)